.github/workflows/test-coding-standards.yml

```diff
@@ -25,8 +25,8 @@ jobs:
       fail-fast: false
       matrix:
         php-version:
-          - '7.4'
           - '8.0'
+          - '8.1'
```

Comment:
I think we only need one 8.x check - we can probably drop this matrix altogether.

Reply:
Done.